src/main/java/io/pivotal/migration/MigrationClient.java

```diff
@@ -745,6 +746,13 @@ else if ("pending".equals(status)) {
 					importedIssue.setFailure("No URL for imported issue: " + body);
 					return false;
 				}
+				var jiraResolution = jiraIssue != null ? jiraIssue.getFields().getResolution() : null;
+				if (jiraResolution != null && RESOLUTION_TYPES_FOR_NOT_PLANNED_MAPPING.contains(jiraResolution.getName())) {
+					boolean success = updateStateReasonToNotPlanned(jiraIssue, url);
+					if (!success) {
+						return false;
```

Comment:
Should the whole import process fail?

Reply:
good point. I will change it, so the process will not fail, but we get a warning, so that we can do it manually